File: コーディングによる問題解決/03 アルゴリズム 初級/全探索/01_整数Vはあるか/1.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/);

const N = Number(input[0]);
const V = Number(input[1]);
const A = input.slice(2).map(Number);

for (let i = 0; i<N; i++){
    if(A[i] == V){
        console.log("Yes");
        process.exit();
    }
}

console.log("No")

File: コーディングによる問題解決/03 アルゴリズム 初級/全探索/01_整数Vはあるか/1.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int V = sc.nextInt();
        int[] A = new int[N];
        for (int i = 0; i < N; i++) {
            A[i] = sc.nextInt();
        }

        for (int i = 0; i < N; i++) {
            if (A[i] == V) {
                System.out.println("Yes");
                return;  // exit() の代わり
            }
        }
        System.out.println("No");
    }
}
